config/initializers/aws.rb

```diff
@@ -0,0 +1,9 @@
+require 'aws-sdk'
+require 'aws-sdk-resources'
+
+AWS.config(
+  :access_key_id => ENV['AWS_ACCESS_KEY_ID'],
+  :secret_access_key => ENV['AWS_SECRET_ACCESS_KEY']
+)
+
+S3_BUCKET =  AWS::S3.new.buckets[ENV['S3_BUCKET']]
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.